pkg/cli/newapp/newapp.go

```diff
@@ -458,7 +458,7 @@ func (o *AppOptions) RunNewApp() error {
 			}
 		case *buildv1.BuildConfig:
 			if t.Spec.Strategy.Type == buildv1.JenkinsPipelineBuildStrategyType {
-				fmt.Fprintf(o.Action.ErrOut, "JenkinsPipeline build strategy is deprecated. Use Jenkinsfiles directly on Jenkins or OpenShift Pipelines instead")
+				fmt.Fprintf(o.Action.ErrOut, "JenkinsPipeline build strategy is deprecated. Use Jenkinsfiles directly on Jenkins or OpenShift Pipelines instead\n")
```

Comment:
Maybe just use `fmt.Fprintln` instead of manually adding the newline.

Reply:
@soltysh Updated it, thanks